build.project.xml

```diff
@@ -467,6 +506,6 @@
     <target
         name="install-dev"
         description="Install the website and set up the development environment."
-        depends="setup-virtuoso-permissions, purge-solr-backend, purge-virtuoso-backend, clean-redis-settings, install, ensure-search-api-solr-site-hash, setup-redis, import-rdf-fixtures, setup-dev, configure-apache-solr-drupal, configure-matomo-drupal, create-demo-users" />
+        depends="setup-virtuoso-permissions, purge-solr-backend, purge-virtuoso-backend, clean-redis-settings, disable-webtools-geocoding-cache, install, ensure-search-api-solr-site-hash, setup-redis, import-rdf-fixtures, setup-dev, configure-apache-solr-drupal, configure-matomo-drupal, create-demo-users, enable-webtools-geocoding-cache" />
```

Comment:
I don't think persisting the cache should be the default setup when installing. I did some manual tests and I ended up by Git status reporting a lot of new files. In th every rare cases when developer wants to create some Webtools cached items, he will manually enable the feature. However, I don't believe that we should go with this "persist cache switch" -- see bellow.

Reply:
OK we can add this to the test pipeline.

Reply:
I have tried removing it but the results are not ideal. Without this every time a test is started it will abort immediately and inform that the Phing target needs to be executed first. Unfortunately this needs to be active for every test run otherwise we cannot detect if any requests were made during the test.

Wouldn't it be better to leave this in? Writing tests is a big part of the development work and I think it would become tiresome to execute this manually after every build.